test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -237,13 +285,24 @@ private void CopyLatestCliToTestDirectory(string destinationDir)
         private void UpdateCliWithLatestNuGetAssemblies(string cliDirectory)
         {
             var nupkgsDirectory = DotnetCliUtil.GetNupkgDirectoryInRepo();
+#if NETCORE3_0
             var pathToPackNupkg = FindMostRecentNupkg(nupkgsDirectory, "NuGet.Build.Tasks.Pack");
+#else
+            var pathToPackNupkg = FindMostRecentNupkg(nupkgsDirectory, "NuGet.Build.Tasks.Pack.Sdk2x");
+#endif
 
             var nupkgsToCopy = new List<string> { "NuGet.Build.Tasks", "NuGet.Versioning", "NuGet.Protocol", "NuGet.ProjectModel", "NuGet.Packaging", "NuGet.LibraryModel", "NuGet.Frameworks", "NuGet.DependencyResolver.Core", "NuGet.Configuration", "NuGet.Common", "NuGet.Commands", "NuGet.CommandLine.XPlat", "NuGet.Credentials" };
 
-            var pathToSdkInCli = Path.Combine(
-                    Directory.GetDirectories(Path.Combine(cliDirectory, "sdk"))
-                        .First());
+            var sdkPaths = Directory.GetDirectories(Path.Combine(cliDirectory, "sdk"));
+
+#if NETCORE3_0
+            var pathToSdkInCli = sdkPaths.Where(path => path.Split(Path.DirectorySeparatorChar).Last().StartsWith("3")).First();
+#else
+            var pathToSdkInCli = sdkPaths.Where(path => path.Split(Path.DirectorySeparatorChar).Last().StartsWith("2")).First();
+#endif
+          //  var pathToSdkInCli = Path.Combine(
```

Comment:
No uncommented code please :)

Reply:
Do you think remove it in another PR of removing the second N.B.T.P package will be better?
As we don't have to patch in msbuildIntegratedTestFixture, we may need to change everything back. So keeping those will be helpful when we change back.

Reply:
You can leave it then I guess.